scripts/maverick.js

```diff
@@ -0,0 +1,7 @@
+function myDetails(name, id ,email,language, stage) {
+   return `Hello World, this is ${name} with HNGi7 ID ${id} and email ${email} using ${language} for stage ${stage} task`
+ };
+ myDetails('Edidiong Ekpo', 'HNG-02187','eddymav247@gmail.com', 'javaScript', '1');
```

Comment:
The stage is suppose to be set to 2. See [here](https://github.com/hng-teamfury/team-fury-1/blob/dev/README.md#the-goal) for an example.